tests/unit/test_footprinter.cc

```diff
@@ -474,48 +515,21 @@ TEST_F(TestFootprinter, test_vector) {
       sizeof(v)
     );
   }
-}
-
-struct TestBase {
-
-  checkpoint_virtual_serialize_root()
-
-  virtual ~TestBase() = default;
-
-  template <typename SerializerT>
-  void serialize(SerializerT& s) {
-    s | a;
-  }
-
-private:
-  int a = 0;
-};
-
-struct TestDerived2 : TestBase {
-  explicit TestDerived2(int i) {}
-  explicit TestDerived2(SERIALIZE_CONSTRUCT_TAG) {}
 
-  checkpoint_virtual_serialize_derived_from(TestBase)
+  {
+    std::vector<TestDerived2> v;
+    EXPECT_GE(
+      checkpoint::getMemoryFootprint(v),
+      sizeof(v)
+    );
```

Comment:
I think this test could still fail spuriously on some platforms, if a default-constructed `vector<T> v` has `v.capacity() > 0`.

Reply:
`EXPECT_GE` (greater or equal) should cover this

Reply:
Oops, sorry, I misread that this says `EXPECT_GE` now.